src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +80,60 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+/**
+ * Represents a Person's address's block number in the address book.
+ */
+class Block {
+    private int blockNumber;
```

Comment:
You should use getters and setters to for storing and retrieving values from class members. The constructor is ideally for initialization operations only.